packages/actor-abstract-path/lib/ActorAbstractPath.ts

```diff
@@ -205,10 +205,11 @@ export abstract class ActorAbstractPath extends ActorQueryOperationTypedMediated
      */
   // Let the iterator `it` emit all bindings of size 2, with subjectStringVariable as value subjectVal
   // and objectStringVariable as value all nodes reachable through predicate* beginning at objectVal
+  /* eslint max-len: off */
```

Comment:
This can probably go if the next diff is undone.

Reply:
Done!